rs/nns/governance/tla/Merge_Neurons.tla

```diff
@@ -0,0 +1,350 @@
+---- MODULE Merge_Neurons ----
+EXTENDS TLC, Sequences, Integers, FiniteSets, Variants
+
+(*
+@typeAlias: proc = Str;
+@typeAlias: account = Str;
+@typeAlias: neuronId = Int;
+@typeAlias: methodCall = Transfer({ from: $account, to: $account, amount: Int, fee: Int}) | AccountBalance({ account: $account });
+@typeAlias: methodResponse = Fail(UNIT) | TransferOk(UNIT) | BalanceQueryOk(Int);
+@typeAlias: neurons = $neuronId -> {cached_stake: Int, account: $account, maturity: Int, fees: Int};
+*)
+
+_type_alias_dummy == TRUE
+
+CONSTANTS
+    Minting_Account_Id,
+    Merge_Neurons_Process_Ids,
+    TRANSACTION_FEE
+
+TRANSFER_OK == "Ok"
+TRANSFER_FAIL == "Err"
+DUMMY_ACCOUNT == ""
+
+\* @type: ($neurons, $neuronId) => Int;
+Minted_Stake(neuron, neuron_id) == neuron[neuron_id].cached_stake - neuron[neuron_id].fees
```

Comment:
Perhaps it would be better to call this something like `Effective_Stake`. `Minted_Stake` makes me think it has to do with what's minted on the ledger, which this is not.